client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -122,7 +123,9 @@ const DomainPicker: FunctionComponent< Props > = ( {
 		}
 	}, [ allSuggestions ] );
 
-	useTrackModal( 'DomainPicker' );
+	useTrackModal( 'DomainPicker', () => ( {
+		selected_domain: onboardingStore.getSelectedDomain(),
```

Comment:
`getSelectedDomain` looks like it returns an object ?

<img width="390" alt="Screen Shot 2020-04-24 at 6 45 49 pm" src="https://user-images.githubusercontent.com/6458278/80193276-da401480-865b-11ea-8ebf-0063e08e3e2c.png">

Reply:
oops, good catch